.travis.yml

```diff
@@ -2,6 +2,11 @@ language: php
 
 php: [5.3, 5.4, 5.5, 5.6, hhvm]
 
+matrix:
+  include:
+    - php: 5.6
+      env: ZOMBIE_VERSION='@~2.0'
```

Comment:
Maybe I've misunderstood the changes you've made. So you're running:
* all PHP versions against Zombie 3.x
* PHP 5.6 version against Zombie 2.x?

If that is so, then all is fine.